web/crux-ui/tailwind.config.js

```diff
@@ -1,52 +1,52 @@
 module.exports = {
-    content: [
```

Comment:
The *.js formatting rule is missing for indents from editorconfig, please add that too to avoid future re-formatting.

Reply:
Added.